src/sass/pages/cashier.scss

```diff
@@ -105,7 +105,15 @@
     width: 111px;
     height: 38px;
 }
-
+.button_cashier {
```

Comment:
It was just to make the buttons ('Deposit' & 'Withdraw') having the same width.
I removed the class.